nuxt.config.js

```diff
@@ -1,26 +1,25 @@
 // Contentful + Environment variables
-const api = require('./.contentful.json');
 
 // These allow us to query Contentful and get all of our valid URLs.
 const contentful = require('contentful');
 const client = contentful.createClient({
-  host: api.CTF_HOST,
-  space: api.CTF_SPACE_ID,
-  environment: api.CTF_ENVIRONMENT,
-  accessToken: api.CTF_CDA_PREVIEW_TOKEN || api.CTF_CDA_ACCESS_TOKEN,
+  host: process.env.CTF_HOST,
+  space: process.env.CTF_SPACE_ID,
+  environment: process.env.CTF_ENVIRONMENT,
+  accessToken: process.env.CTF_CDA_PREVIEW_TOKEN || process.env.CTF_CDA_ACCESS_TOKEN,
 });
 
 module.exports = {
   //
   // Environment variables
   //
   env: {
-    CTF_HOST: api.CTF_HOST || 'cdn.contentful.com',
-    CTF_SPACE_ID: api.CTF_SPACE_ID,
-    CTF_ENVIRONMENT: api.CTF_ENVIRONMENT || 'master',
-    CTF_CDA_ACCESS_TOKEN: api.CTF_CDA_ACCESS_TOKEN,
-    CTF_CDA_PREVIEW_TOKEN: api.CTF_CDA_PREVIEW_TOKEN,
-    baseUrl: api.BASE_URL || 'https://reports.unocha.org',
+    CTF_HOST: process.env.CTF_HOST || 'cdn.contentful.com',
+    CTF_SPACE_ID: process.env.CTF_SPACE_ID,
+    CTF_ENVIRONMENT: process.env.CTF_ENVIRONMENT || 'master',
```

Comment:
Some of the defaults (although fake) are already provided in the Dockerfile. The `CTF_ENVIRONMENT` for example.
You decide where do you want to put them but lets group them up in either one.

Reply:
Keeping them in docker makes sense to me. I'll throw one more commit up here to remove all the JS "defaults"